contracts/Comet.sol

```diff
@@ -589,9 +589,11 @@ contract Comet is CometMainInterface {
 
     /**
      * @dev The change in principal broken into repay and supply amounts
-     * @dev Note: The assumption `newPrincipal >= oldPrincipal` MUST be true
      */
     function repayAndSupplyAmount(int104 oldPrincipal, int104 newPrincipal) internal pure returns (uint104, uint104) {
+        // If the new principal is less than the old principal, then no amount has been repaid or supplied.
```

Comment:
unbelievable nit but are these the only full-sentence comments in the contract?

Reply:
actually they aren't. I'll clean it up in a separate PR (if at all)

Reply:
Sure thing, I removed the `.` here